bin/server

```diff
@@ -261,7 +264,8 @@ datastore.setup(datastoreConfig.ddlSyncEnabled).then(() =>
         queueWebhook: {
             executor,
             queueWebhookEnabled
-        }
+        },
+        unzipArtifactsEnabled
```

Comment:
should this flag be also into `feature.unzipArtifactsEnabled`?

Reply:
I think this is fine because `config.unzipArtifactsEnabled` is used when calling `lib/server.js`.

https://github.com/screwdriver-cd/screwdriver/pull/2637/files#diff-1b48bb429de959672dadab9c87c215e19177dd70e3bd932d131ce74294c44cd5R142

Reply:
yes, this comment is kind of refactoring comment. 
I wonder the objects schemas are different, it might cause a little confusing.